cycledash/static/js/runs/components/forms.js

```diff
@@ -93,30 +100,37 @@ var NewBAMForm = React.createClass({
   },
   render: function() {
     return (
-      <form method='POST' action='/api/bams' className='bam-form' ref='bamForm'>
-        <h3>New BAM</h3>
-
-        <TextInput label='Name:' name='name' required={true}
-                   placeholder='...' />
-        <TextInput label='Tissues:' name='tissues'
-                   placeholder='Left Ovary' />
+      <form method='POST' action='/api/bams' className='add-form' ref='bamForm'>
+        <div className='row'>
+          <h3>New BAM</h3>
+          <div className='add-form-input-half'>
```

Comment:
Can we add the .add-form-input-half style to the underlying TextInput instead of wrapping each TextInput with this div?